test/built-ins/DataView/prototype/setFloat32/to-boolean-littleendian.js

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-dataview.prototype.setfloat32
+es6id: 24.2.4.13
+description: >
+  Boolean littleEndian argument coerced in ToBoolean
+info: |
+  24.2.4.13 DataView.prototype.setFloat32 ( byteOffset, value [ , littleEndian ] )
+
+  1. Let v be the this value.
+  2. If littleEndian is not present, let littleEndian be false.
+  3. Return ? SetViewValue(v, byteOffset, littleEndian, "Float32", value).
+
+  24.2.1.2 SetViewValue ( view, requestIndex, isLittleEndian, type, value )
+
+  ...
+  15. Let bufferIndex be getIndex + viewOffset.
+  16. Return SetValueInBuffer(buffer, bufferIndex, type, numberValue, isLittleEndian).
+
+  24.1.1.6 SetValueInBuffer ( arrayBuffer, byteIndex, type, value [ , isLittleEndian ] )
+
+  ...
+  8. If type is "Float32", then
+    a. Set rawBytes to a List containing the 4 bytes that are the result of
+    converting value to IEEE 754-2008 binary32 format using “Round to nearest,
+    ties to even” rounding mode. If isLittleEndian is false, the bytes are
+    arranged in big endian order. Otherwise, the bytes are arranged in little
+    endian order. [...]
+  ...
+features: [DataView.prototype.getFloat32, Symbol]
+---*/
+
+var buffer = new ArrayBuffer(4);
+var sample = new DataView(buffer, 0);
+
+// False
+sample.setFloat32(0, 1);
+assert.sameValue(sample.getFloat32(0), 1, "no arg");
+sample.setFloat32(0, 2, undefined);
+assert.sameValue(sample.getFloat32(0), 2, "undefined");
+sample.setFloat32(0, 3, null);
+assert.sameValue(sample.getFloat32(0), 3, "null");
+sample.setFloat32(0, 4, 0);
+assert.sameValue(sample.getFloat32(0), 4, "0");
+sample.setFloat32(0, 5, "");
+assert.sameValue(sample.getFloat32(0), 5, "the empty string");
+
+// True
+sample.setFloat32(0, 6, {});
+assert.sameValue(sample.getFloat32(0, true), 6, "{}");
```

Comment:
changing it here does not solve a similar - but bigger - problem in `set-values-return-undefined.js` where it depends on the getFloat32's second parameter as well.

These set tests depend a lot on their respective `DataView#get` methods, I don't like this but it was the best approach I found. At least the get methods does not depend on any set methods rather than `setUint8`.

I'll change it in this file as suggested, but I don't thing it helps considering the other test file I mentioned above.